build/pulsar_ci_tool.sh

```diff
@@ -154,6 +154,61 @@ function ci_move_test_reports() {
   )
 }
 
+function ci_check_ready_to_test() {
+  if [[ -z "$GITHUB_EVENT_PATH" ]]; then
+    >&2 echo "GITHUB_EVENT_PATH isn't set"
+    return 1
+  fi
+  # check ready-to-test label
+  if ! jq -e '.pull_request.labels[] | .name | select(. == "ready-to-test")' "$GITHUB_EVENT_PATH" &> /dev/null; then
```

Comment:
```suggestion
  # check ready-to-test label
  if jq -e '.pull_request.labels[] | .name | select(. == "ready-to-test")' "$GITHUB_EVENT_PATH" &> /dev/null; then
    return 0
  fi
```

I'd say the first time I don't notice we check the label and return on matched here. So I'd recall this suggestion. But it should not be a blocker.